docs/product-release-notes/src/main/asciidoc/rn-rhpam-701-con.adoc

```diff
@@ -0,0 +1,58 @@
+[id='rn-rhpam-701-con']
+= Release notes for {PRODUCT} 7.0.1
+
+{PRODUCT} 7.0.1 provides increased stability and fixed issues listed in this section.
+
+== {CENTRAL}
+
+* Updating task details alters Due On time [https://issues.jboss.org/browse/RHPAM-1229[RHPAM-1229]]
```

Comment:
This issue is not (yet) fixed in 7.0.1 - it was reported as a regression RHPAM 7.0.1. Unless we get a rebuild with this fix included (not likely, because the fix version was set to 7.0.2) this should probably be removed.